pkg/bindings/connection.go

```diff
@@ -113,13 +113,12 @@ func (c *Connection) DoRequest(httpBody io.Reader, httpMethod, endpoint string,
 		err      error
 		response *http.Response
 	)
-	safePathValues := make([]interface{}, len(pathValues))
+	var safePathValues []interface{}
```

Comment:
If you keep make(), then use for loop iteration value, eg:

safePathValue[i] = url.QueryEscape(pv)

Then you don't need to append.  Shaves some time off.